modules/snooker/src/test/ts/browser/EraseOperationsTest.ts

```diff
@@ -580,7 +580,7 @@ UnitTest.test('EraseOperationsTest', () => {
   };
   Assertions.checkDelete(
     'TBA',
-    Optional.some({ section: 0, row: 0, column: 0 }),
+    Optional.some({ section: 0, row: 2, column: 0 }),
```

Comment:
Sanity check: based on the two points in your description, this tests the first (going to previous instead of first cell), and the test cases further down deal with the second point (cef cells)?

If yes, do we care to add a deleting rows case too for the first point? Or is that already covered in another case? I looked up and down a bit, but can't see a specifically relevant case for it.

Reply:
It's covered in another case (you can see I had to change one test from row 0 to row 2).

Reply:
Discussed in a call and clarified. I'll make sure we have a case for both row/column deletion.

Reply:
Have added the extra test case